### ISSUE TITLE
Sell Chest Error / ExcellentShop

### ISSUE THREAD

**Issue by InvisibleSpiders** [BG-Software-LLC/WildChests#246]
### Minecraft's Version

Paper 1.21 Build 130

### Plugin's Version

2024.3

### Describe the bug

Sell chests generate this error : 

https://mclo.gs/hWAuoVA

### To Reproduce

Install both ExcellentShop/NiteCore 2.6.3, and recent WildChests build.
Place sell chest and use a hopper or alt method to insert items and allow to sell.
See console.

**Comment by InvisibleSpiders** [comment]
I can confirm this issue is present within 1.21.1 paper build as well.

**Comment by OmerBenGera** [comment]
I am using nightcore v2.6.3, ExcellentShop v4.11.5 and WildChests v2024.3.57 and can't replicate the issue myself.

**Comment by InvisibleSpiders** [comment]
So I'm not sure if this was due to a server version upgrade, but existing sell chests were generating that error.
I generated new chests via the command and replaced them, which solved the issue.

I did note that the affected chests weren't selling items 90% of the time and particles weren't working.
Once they were replaced, all functions continued to work as normal.

I suppose this can get archived then. Thanks for looking into it! :)
